pplns-with-job-declaration.tex

```diff
@@ -21,174 +21,193 @@
 
 \section{Introduction}
 
-Right now, payout schemes for miners account only for provided hash power.
-One of the biggest bitcoin issue is the centralization of transaction selection, this issue is solved by Stratum v2 \cite{sv2}. Given that Stratum v2 let the miner select their own transaction (this feature is known as \emph{Job Declaration}), and that transactions fees might become the most important part of the mining reward, we need a system that calculate the payouts based both on selected transactions and hash power provided. We propose a new payment schema, called PPLNS-JD, that expands PPLNS (see \cite{rosenfeld} and \cite{ocean}), so that miner will get paid in a fair way given the hash power provided, and the fees in the mined job. The core idea is that a miner perfoming Job Declaration who is more capable to find the best set of transactions in terms of fee over vbytes (and consequentially in fees of the block template), should be rewarded more. PPLNS-JD can be implemented as an extension of Sv2 \cite{extension} and it is easly accountable, so that miners' trust in pool operator is minimized. 
+Current payout schemes for mining pools account only for provided hashrate, which is not ideal for Job Declaration.
 
-We define a fair payment schema that splits the share payout in two parts. The first comes from the coinbase and pays accordingly to the difficulty score of the share, in the same fashion as PPLNS. The second comes from the block fee and pays accordigly to share difficulty score and the share fee score in such a way that share with the highest fee in its group of shares (slice, see below) is paid at least $F_S/n$, where $F_S$ is the part of the block fee reserved to that group of shares $S$ and $n$ is the number of shares in $S$.
+One of the biggest issues for the Bitcoin network is the centralization of transaction selection, this issue is solved by Stratum v2 \cite{sv2}. Stratum v2 enables a miner to select their own block template of transactions (this feature is known as \emph{Job Declaration}). Since transactions fees will become the largest percentage of the block reward, the network needs a system that calculates payouts based both on selected transaction fees and hashrate provided. We propose a new payment schema, called PPLNS-JD (or informally Job Slicing), that expands PPLNS (see \cite{rosenfeld} and \cite{ocean}), so a miner can be paid fairly given hashrate provided, and fees in the mined block template. The core idea is that a miner performing Job Declaration, who are more capable of optimizing transaction selection for fee over vbytes (and consequentially in fees of the block template), should be rewarded more. PPLNS-JD can be implemented as an extension of Sv2 \cite{extension} and it is easily accountable, so that miners' trust in pool operator is minimized.
```

Comment:
Why Job Slicing? Do you have any motivation for changing naming it like this? 
JD stands for Job Declaration, so if we call it PPLNS-JD sounds strange to refer it to Job Slicing

Reply:
Couldn't think of a shorter name that was relevant. Saying "PPLNS-JD" is undesirable though. I am open to other names but thought it reasonable to add something here as an example. I'm also fine with omitting a short name in the text here.

Reply:
Share Slicing?
Because the shares are sliced into groups.

Reply:
It is a variation of Pay Per Last N Shares, so it must contain PPLNS in the name, IMO